src/openms/source/ANALYSIS/RNPXL/RNPxlReport.cpp

```diff
@@ -267,18 +267,7 @@ namespace OpenMS
             {
               AASequence new_aa = AASequence::fromString("[" + String(rna_weight) + "]." + seq);
               ph.setSequence(new_aa);
-            }
-            else // place it anywhere
-            {
-              for (auto a : aa) // NOTE: performs copy, this cannot possibly work!
-              {
-                if (!a.isModified())
-                {
-                  a.setModification("[" + String(rna_weight) + "]");
-                  break;
-                }
-              }
-            }
```

Comment:
ok, so this never did anything, so removing it is safe, I guess.
But *should* it do something, i.e. is it actually a bug right now?

Reply:
will get rid of the RNPxlReport anyway soon so this is not really relevant anymore

Reply:
ok. Makes sense.
PR looks good to me